openmicroscopy/bioformats
# Roi

Do not store the z, c, t on the shape if no position set.
To test:

test 1:
 * Open an image with several z/t using B-F importer e.g. ```dv/IAGFP-Noc01_R3D.dv```
 * Draw a shape and add it to the ROI manager
 * Check that the position is not set (ROIManager>properties)
 * Export using B-F exporter
 * Open the generated OME-TIFF and check the OME-XML section of it. You should not have the following attributes ```TheC```, ```TheZ``` ```TheT```

test 2:
 * Repeat the same steps but this time, set a position e.g. 1, 3, 4 (depends on the image selected)
 * Export using B-F exporter
 * Open the generated OME-TIFF and check the OME-XML section of it. You should have the following attributes ```TheC=0```, ```TheZ=2``` ```TheT=3```
